Cabal/Distribution/Package.hs

```diff
@@ -69,6 +70,44 @@ import Distribution.ModuleName
 
 import Text.PrettyPrint ((<+>), text)
 
+-- | A Component name, or other similarly-parsed identifier.
```

Comment:
Distinguished from a 'ComponentName', which also identifies whether or not the component in question is a library, executable, etc.